src/vs/workbench/contrib/debug/common/debugModel.ts

```diff
@@ -1436,7 +1436,8 @@ export class DebugModel implements IDebugModel {
 				}
 			}
 		});
-		this.exceptionBreakpoints.forEach(ebp => {
+		this.exceptionBreakpoints.forEach(iebp => {
+			const ebp = iebp as ExceptionBreakpoint;
```

Comment:
It's ok for `this.exceptionBreakpoints` to be the real `ExceptionBreakpoint` class, that seems to be the case for other properties

Reply:
The real ExceptionBreakpoint is defined in model and though exported, is generally not used in debug service or session. Since currently model creates ExceptionBreakpoint objects, and only model or view needs to use these objects. Session is only holding IExceptionBreakpoint.

With the current design, it is either a "type cast" here or in setExceptionBreakpoint method. Open to suggestions if we should be doing something else.